app/javascript/lib/scp-api.js

```diff
@@ -120,6 +120,67 @@ export function setMockOrigin(origin) {
   mockOrigin = origin
 }
 
+/** Constructs and encodes a URL parameter, if it has a value */
+function encodeParam(param, value, isFirst=false) {
+  const delimiter = isFirst ? '?' : '&'
+  if (!value || value === '') {
+    // Omit empty parameters in API request
+    return ''
+  } else {
+    return `${delimiter}${param}=${encodeURIComponent(value)}`
+  }
+}
+
+/**
+* Returns initial content for the "Explore" tab in Study Overview
+* @param {String} studyAccession Study accession
+*/
+export async function fetchExploreInitialization(studyAccession, mock=false) {
+  const apiUrl = `/studies/${studyAccession}/explore`
+  const [exploreInit, perfTime] =
+    await scpApi(apiUrl, defaultInit(), mock, false)
+
+  return exploreInit
+}
+
+/**
+ * Returns an object with scatter plot data for a cluster in a study
+ *
+ * This endpoint is volatile, so intentionally not documented in Swagger.
```

Comment:
this endpoint is now documented in swagger :)

Reply:
Thanks, [updated](b24e07f).